packages/experimental/MenuButton/src/__tests__/__snapshots__/MenuButton.test.win32.tsx.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ContextualMenu default 1`] = `
+<View
+  accessibilityLabel="Press for Nested MenuButton"
+  accessibilityRole="button"
+  accessible={true}
+  focusable={true}
+  onAccessibilityTap={[Function]}
+  onBlur={[Function]}
+  onClick={[Function]}
+  onFocus={[Function]}
+  onKeyUp={[Function]}
+  onPress={[Function]}
+  onResponderGrant={[Function]}
+  onResponderMove={[Function]}
+  onResponderRelease={[Function]}
+  onResponderTerminate={[Function]}
+  onResponderTerminationRequest={[Function]}
+  onStartShouldSetResponder={[Function]}
+  style={
+    Object {
+      "alignItems": "center",
+      "alignSelf": "flex-start",
+      "backgroundColor": "#f3f2f1",
+      "borderColor": "#8a8886",
+      "borderRadius": 2,
+      "borderWidth": 1,
+      "display": "flex",
+      "flexDirection": "row",
+      "justifyContent": "center",
+      "minHeight": 32,
+      "minWidth": 80,
+      "paddingEnd": 16,
+      "paddingStart": 16,
+      "width": undefined,
+    }
+  }
+>
+  <Text
+    style={
+      Object {
+        "color": "#323130",
+        "fontFamily": "Segoe UI",
+        "fontSize": 12,
+        "fontWeight": "400",
+        "margin": 0,
+        "marginLeft": 0,
+      }
+    }
+  >
+    Press for Nested MenuButton
+  </Text>
+</View>
```

Comment:
I'm curious why the chevron icon doesn't seem to show up in the tree

Reply:
Maybe because it's as SVG file here. I'll make it inline as in the old MenuButton